RELEASE_NOTES.md

```diff
@@ -4,6 +4,8 @@
 
 ## General
 
+* Changed download logic to allow arm64 RKE2 CNI installation tarballs
```

Comment:
I'd suggest a different wording. The target audience for these notes is usually not familiar with the codebase. Something along the lines of "Allow RKE2 deployments with Calico, Cilium and Multus on aarch64 platforms" would be more suitable.